### PR TITLE
Add base for styles and working on styles

### PR DESCRIPTION
**Pull request by agjohnson**
Styles are not complete, but all of the base pieces are in the client
here and server side -- ad_types, etc.

* Change to style-loader for styles
* Change attribute names
* Add base for styles to allow for testing
* Uses templates added for ad types on prod server
* Uses versioned templates

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -91,7 +86,7 @@ export class Placement {
  */
 export function load_placements() {
   // Find all elements matching required data binding attribute
-  const node_list = document.querySelectorAll("[data-ethical-ad-publisher]");
+  const node_list = document.querySelectorAll("[data-ea-publisher]");
```
Comment (ericholscher) [inline index.js:89]: These selectors should probably be constants as well.

**Comment by agjohnson** [comment]
I'd definitely like to see some sort of page with the ad displays documented in a nice way for marketing folks. For now, I can at least screenshot everything in the readme.

I've updated the styles to fix a few issues:

image

The changes include reducing the size of the placements, moving the callout to outside the placements, and i'm trying a little bit of color for `<b>` tags.

**Comment by agjohnson** [comment]
Single line looks quite a bit better now as well:
image

File: styles.scss [hunk 2]
```diff
@@ -1,12 +1,208 @@
 /* Ad styles */
 
-[data-ethical-ad-publisher] {
-  background: #f0f0e0;
-  font-family: monospace;
-  margin: 1rem;
-  padding: 1rem;
+// Sizes
+$width-image-horizontal: 320px;
```
Comment (davidfischer) [inline styles.scss:4]: All the existing ads in our system are 240x180 and displayed at 120x90.

Comment (agjohnson) [reply styles.scss:4]: This is not the image width, this is the bounding box width

**Comment by agjohnson** [comment]
I copied this from the rtd ad type template. I'm not tied to "ads served ethically" and had the same issue with the copy. It didn't seem to function super well as a branding mechanism either. For example "Ad by EthicalAds" might solve both issues. I'll take this conversation to chat.

**Comment by agjohnson** [comment]
I've updated the templates with copy change and the styleguide test page as well.

**Pull request merged**